### ISSUE TITLE
REPL Errors do not read their contents

### ISSUE THREAD

**Issue by schanzer** [ds26gte/code.pyret.org#86]
When evaluating `"moo` (unclosed string), the REPL shows a nicely-descriptive error. However, the screen reader does not read the full text of the error, instead reporting `"moo resulted in an error`.

As a rule, we should not hide information from Visually-Impaired programmers. That means reading the error in its entirety.

**Comment by ds26gte** [comment]
Fixed in 7208e21.
